uuid.go

```diff
@@ -36,11 +36,6 @@ func (u *UUID) DeepCopyInto(out *UUID) {
 	out.UUID = (uuid.UUID)(o)
 }
 
-// Equal returns true if itself is equal to other.
-func (u UUID) Equal(other UUID) bool {
-	return uuid.Equal(u.UUID, other.UUID)
-}
-
```

Comment:
I don't think, it's that simple.

Reply:
why ?

Reply:
The embed struct seems to be treated like the type alias. :+1: